documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,183 @@
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-2.png "Jenkins New Item")
+   </p>  
+
+1. Choose **Save**. 
+
+1. To run your pipeline, choose **Build Now** in the job UI.
+
+
+## Add a Build Step
+
+1. In your `Jenkinsfile`, add the following code snippet: 
+   ```
+    stage('build') {
+      mtaBuild script: this
+    }
+   ```
+   **Result:** The `mtaBuild` step calls a build tool to build a multi-target application (MTA). The tool consumes an MTA descriptor that contains the metadata of all entities which comprise an application or are used by one during deployment or runtime, and the dependencies between them. For more information about MTAs, see [sap.com][sap]. 
+   
+2. Create an MTA descriptor with the name `mta.yaml`, which contains the following code:
+   ```
+    _schema-version: 2.1.0
+    ID: com.sap.piper.node.hello.world
+    version: 1.0.0
+    description: A Hello World sample application
+    provider: SAP Sample generator
+    modules:
+      - name: piper.node.hello.world
+        type: nodejs
+        path: .
+   ```
+   
+3. To configure the step to build an MTA for the Cloud Foundry environment, in your repository, open or create the `.pipeline/config.yml` and add the following content: 
+   ```
+    general:
+    steps:
+      mtaBuild:
+        buildTarget: 'CF'
+   ```
+   For additional information about the configuration, have a look at the [Common Configuration Guide][resources-configuration] and the [MTA build step documentation][resources-step-mtabuild].
+
+4. Commit your changes.
+
+5. To run your pipeline, choose **Build Now** in the job UI.
+
+## Add a Deploy Step
+
+1.  In your `Jenkinsfile`, add the following code snippet:
+   ```
+   stage('deploy') {
+     def mtarFilePath = commonPipelineEnvironment.getMtarFilePath()
+     cloudFoundryDeploy( script: this, mtaPath: mtarFilePath)
+   }
+   ```
+   **Result:** The `cloudFoundryDeploy`  step calls the Cloud Foundry command line client to deploy into SAP Cloud Platform.
+
+2. To configure the step to deploy into the Cloud Foundry environment, in your repository, open or create the `.pipeline/config.yml` and add the following content:
+   ```
+    cloudFoundryDeploy:
+      deployTool: 'mtaDeployPlugin'
+      deployType: 'standard'
+      cloudFoundry:
+        org: '<your-organisation>'
+        space: '<your-space>'
+        credentialsId: 'CF_CREDENTIALSID'
+   ```
+   The key `CF_CREDENTIALSID` refers to a user-password credential you must create in Jenkins: In Jenkins, choose **Credentials** from the main menu and add a **Username with Password** entry.
+   <p align="center">
+   ![Add Credentials](../images/JenkinsCredentials-1.png "Add Credentials")
+   </p>  
+   For more information about the configuration, see the [Common Configuration Guide][resources-configuration] and [cloudFoundryDeploy][resources-step-cloudFoundryDeploy].
+
+3. Commit your changes.
```

Comment:
See my other comment above about commit. User works either in the github web ui than it is a save rather than a commit (technically it is a commit, of course). In case the user works with a local clone it is a commit and push.

Reply:
i go for "Save your changes to your remote repository."